doc/tutorials/quantization.md

```diff
@@ -0,0 +1,60 @@
+# Post-Training Quantization
```

Comment:
Thanks for writing this all up.  I think I wrote this in an earlier comment, but my vision here is to leave this tutorial as-is for now, not link to it from anywhere, and then when the NN documentation comes together, add this as a subsection to the NN documentation instead of a standalone tutorial.  Perhaps eventually as we add more quantization strategies it may make sense to split quantization documentation into its own file, but for now that should work.